cmdline/command.go

```diff
@@ -19,6 +19,7 @@ var commands = []command{
 
 	{"exi[t]", event.Quit},
 	{"q[uit]", event.Quit},
+	{"q!", event.ForceQuit},
```

Comment:
I think it's better to support `q[uit]!` but I remember I didn't implement the parser to accept this so I'll fix this after merging.

Reply:
Yes, I tried _q[uit]!_ as well, but failed due to the parsing. Do you want me to add a failing test case for _q[uit]!_ ?

Reply:
No, thanks. I'll take care of it.